docs/articles/roadmap/README.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: faq
+title: Roadmap
+permalink: /roadmap/
+---
+# Roadmap
+
+This roadmap outlines the most important features currently in our plans or in production.
+
+Please vote on the features you want us to implement first (follow the link and leave a reaction on GitHub). This helps us prioritize and deliver what really matters to you.
+
+Feature                                                                                           | Status
+------------------------------------------------------------------------------------------------- | -------------------
+[Visual Regression Testing](https://github.com/DevExpress/testcafe/issues/1207)                   | Under Consideration
+[Testing in Multiple Browser Windows](https://github.com/DevExpress/testcafe/issues/912)          | Under Consideration
+[Cucumber Integration](https://github.com/DevExpress/testcafe/issues/2835)                        | Under Consideration
+[Selector Debug Panel](https://github.com/DevExpress/testcafe/issues/3244)                        | Under Consideration
+[Entire Page Screenshots](https://github.com/DevExpress/testcafe/issues/1520)                     | Under Consideration
+[Custom Actions](https://github.com/DevExpress/testcafe/issues/1535)                              | Under Consideration
+[Custom TypeScript Compilation Options](https://github.com/DevExpress/testcafe/issues/1845)       | Under Consideration
+[Conditional Browser Skip](https://github.com/DevExpress/testcafe/issues/481)                     | Under Consideration
```

Comment:
IMO it's not pretty clear name, what about `Conditional Test Execution` or `Conditional Test Skip` (not sure). We are going to implement a predicate function for skipping some tests, not only depends on browsers.

Reply:
`Conditional Test Execution` is a good feature name